### PR TITLE
feat: new sample for prometheus rules

### PR DESCRIPTION
**Pull request by howieyuen**
related issue: https://github.com/KusionStack/konfig/issues/45#issuecomment-1288922110

### CODE REVIEW

File: base/examples/monitoring/prometheus-rules/alert/main.k [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import base.pkg.kusion_prometheus.monitoring.v1 as monitoringv1
+
+_alerts: monitoringv1.PrometheusRule {
```
Comment (Peefy) [inline base/examples/monitoring/prometheus-rules/alert/main.k:3]: The variable `_alerts: `  can be omitted if it is not used elsewhere.

File: base/examples/monitoring/prometheus-rules/alert/main.k [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    metadata = {
+        name = "example-alert"
+        namespace = "default"
+        labels: {
+            "prometheus": "example",
```
Comment (Peefy) [inline base/examples/monitoring/prometheus-rules/alert/main.k:8]: Commas "," at end of the line can be omitted.

File: base/examples/monitoring/prometheus-rules/alert/main.k [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            "role": "alert-rules",
+        }
+    }
+    spec = {
+        groups = [
```
Comment (Peefy) [inline base/examples/monitoring/prometheus-rules/alert/main.k:13]: spec.groups = [...]

File: base/examples/monitoring/prometheus-rules/record/main.k [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+) by (cluster)
+"""
+
+# expr: calculates the average rate of increase of node cpu every 5 minutes
+_node_cpu = """\
+sum(rate(node_cpu_seconds_total{job="node-exporter",mode!="idle",mode!="iowait",mode!="steal"}[5m])) /
+count(sum(node_cpu_seconds_total{job="node-exporter"}) by (cluster, instance, cpu))
+"""
+
+_k8s_rules: monitoringv1.PrometheusRule {
```
Comment (Peefy) [inline base/examples/monitoring/prometheus-rules/record/main.k:22]: The variable `_k8s_rules: ` can be omitted if it is not used elsewhere.

File: base/examples/monitoring/prometheus-rules/record/main.k [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    metadata = {
+        name = "k8s-rules"
+        namespace = "default"
+        labels: {
+            "prometheus": "example",
+            "role": "alert-rules",
+        }
+    }
+    spec = {
```
Comment (Peefy) [inline base/examples/monitoring/prometheus-rules/record/main.k:31]: spec.groups = [...]

File: base/examples/monitoring/prometheus-rules/record/main.k [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+sum(rate(node_cpu_seconds_total{job="node-exporter",mode!="idle",mode!="iowait",mode!="steal"}[5m])) /
+count(sum(node_cpu_seconds_total{job="node-exporter"}) by (cluster, instance, cpu))
+"""
+
+_k8s_rules: monitoringv1.PrometheusRule {
+    metadata = {
+        name = "k8s-rules"
+        namespace = "default"
+        labels: {
+            "prometheus": "example",
```
Comment (Peefy) [inline base/examples/monitoring/prometheus-rules/record/main.k:27]: Commas "," at end of the line can be omitted.

**Review by Peefy — APPROVED** [review]
LGTM

**Review by markliby — APPROVED** [review]

**Pull request merged**